liquidity/liquidity.go

```diff
@@ -1020,6 +1035,20 @@ func (m *Manager) currentSwapTraffic(loopOut []*loopdb.LoopOut,
 	return traffic
 }
 
+// refreshAutoloopBudget checks whether the elapsed time since our last autoloop
+// budget refresh is greater than our configured refresh period. If so, the last
+// refresh timestamp.
+func (m *Manager) refreshAutoloopBudget(ctx context.Context) error {
```

Comment:
I'm not sure if I'm missing something, but I don't find where the actual budget is refreshed